types/events.go

```diff
@@ -62,7 +95,8 @@ func init() {
 // but some (an input to a call tx or a receive) are more exotic
 
 type EventDataNewBlock struct {
-	Block *Block `json:"block"`
+	Block   *Block  `json:"block"`
+	BlockID BlockID `json:"block_id"`
```

Comment:
Do we need to backport this as well?

Reply:
It's a good question. I assumed this was in the original PR for some relevant reason, but I didn't check whether it could be moved independently. It wasn't entirely clear which parts of the patch were "important" to the API change, so I assumed they all were.

Reply:
I know this was a request from an external user. However, I don't think it's necessary to have it in 0.34

Reply:
Reverted it.